testng-core/src/test/java/test/listeners/issue2752/ListenerSample.java

```diff
@@ -13,6 +14,7 @@
 public class ListenerSample implements IClassListener, ITestListener {
 
   private static final Map<String, List<String>> logs = new ConcurrentHashMap<>();
+  private static final Map<String, Long> timeLogs = new ConcurrentHashMap<>();
```

Comment:
We could have used a `LinkedHashMap` here so that the insertion order is maintained.